### PR TITLE
Tweaks for neutral latinamerican spanish.

### PR DESCRIPTION
**Pull request by icarito**
Mergear -> Fusionar
Debugguear -> Depurar
Setear -> Definir
...

### CODE REVIEW

**Comment by akien-mga** [comment]
Could you make those changes directly in Weblate which we use to manage Godot's translations? https://hosted.weblate.org/projects/godot-engine/godot/
Otherwise it's hard to keep things in sync on both ends.

File: tools/translations/es.po [hunk 1]
```diff
@@ -1682,7 +1681,7 @@ msgstr "Por Defecto"
 
 #: tools/editor/editor_node.cpp
 msgid "Switch Scene Tab"
-msgstr "Cambiar Pestaña de Escena"
+msgstr "FCambiar Pestaña de Escena"
```
Comment (SuperUserNameMan) [inline tools/translations/es.po]: "FCambiar" ?

Comment (icarito) [inline tools/translations/es.po]: Ouch that was a typo, I just force pushed an amendment.

**Comment by icarito** [comment]
I understand, I looked for a link to translation system such as https://hosted.weblate.org/projects/godot-engine/godot/ but could not find it. I'll propose these changes there then.

**Comment by akien-mga** [comment]
> I looked for a link to translation system such as https://hosted.weblate.org/projects/godot-engine/godot/ but could not find it.

That's right, we got this platform setup only recently and it's not well documented yet. We'll have to fix that.

**Comment by icarito** [comment]
Thanks I updated https://hosted.weblate.org/projects/godot-engine/godot/es/

**Pull request closed without merging**